### DISCUSSION TITLE
Resnet50, 101, 152 problem

### DISCUSSION THREAD

**Discussion by aliotopal** [Trusted-AI/adversarial-robustness-toolbox#1981] [Q&A]
Hi,
I am testing the code with resnet (ResNet50) but it does not give a good result. Is this because of the preprocessing? In the code, preprocessing was (0.5, 0.5) for inceptionV3. Should it be different for resnet?

**Comment by beat-buesser** [comment]
It depends on with which preprocessing the model has been trained. Can you make correct predictions with the ART estimator's method `predict`?
